app.rb

```diff
@@ -0,0 +1,26 @@
+require 'sinatra/base'
+require 'pg'
+
+class Chitter < Sinatra::Base
+    enable :sessions
+    
+    get "/" do
+      
+      erb :index
+    end
+  
+    post "/peeps" do
+    session[:peep] = params[:peep]
```

Comment:
Currently this `post` method is returning the output of this line - in a later iteration of the program it would probably do its action on the database and then you would redirect to another page.